src/App.css

```diff
@@ -18,7 +18,64 @@
   font-size: large;
 }
 
+#text-input {
+  margin-top: 100px;
+}
+
 @keyframes App-logo-spin {
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+ul {
+  list-style: none;
+}
+
+#shopping-cart-logo {
+
+  top: 20px;
+  right: 30px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+}
+
+#shopping-cart-logo:hover {
+  -ms-transform: scale(1.1, 1.1); /* IE 9 */
+   -webkit-transform: scale(1.1, 1.1); /* Safari */
+   transform: scale(1.1, 1.1);
+}
+
+#flyoutPanel {
+    width: 30vw;
+    height: 100vh;
+    background-color: #ECECEC;
+    position: fixed;
+    top: 0;
+    right: 0;
+    transform: translate3d(130vw, 0, 0);
+    transition: transform .3s cubic-bezier(0, .52, 0, 1);
+    overflow: scroll;
+    z-index: 1000;
+}
+
+#close_button {
+  display: block;
+  left: 30px;
+  position: relative;
+  width: 20px;
+  height: 20px;
+  margin-top: 40px;
+}
+
+#close_button:hover {
+  -ms-transform: scale(1.2, 1.2); /* IE 9 */
+   -webkit-transform: scale(1.2, 1.2); /* Safari */
+   transform: scale(1.2, 1.2);
+}
+
+
+#wishlist-header {
+  margin-left: 30px;
+  text-align: left;
+}
```

Comment:
I like this pattern ^ because of what Alex mentions (and any code we find ourselves repeating we can move from the component css to the main app css). But there are lots of great patterns, so if anyone has concerns or other preferences, happy to find one that works for everyone.